app/models/user.js

```diff
@@ -91,6 +105,16 @@ var User = DS.Model.extend({
       }
       return first + ' ' + last;
   }),
+
+  displayName: computed('firstName', 'middleName', 'lastName', {
```

Comment:
displayName can replace fullName.  I don't think we need two ways of printing the users name.